packages/filesystem/src/browser/file-dialog/file-dialog.ts

```diff
@@ -111,12 +112,16 @@ export abstract class FileDialog<T> extends AbstractDialog<T> {
     protected readonly locationListRenderer: LocationListRenderer;
     protected readonly treeFiltersRenderer: FileDialogTreeFiltersRenderer | undefined;
 
+    private readonly treePanel: Panel;
```

Comment:
[question]: Why `private`? The other fields are `protected`.

Reply:
changed